p2p/agent.go

```diff
@@ -108,10 +111,16 @@ type (
 		Self() ([]multiaddr.Multiaddr, error)
 		// ConnectedPeers returns the connected peers' info
 		ConnectedPeers() ([]peer.AddrInfo, error)
+		// ConnectedPeersByNetwork returns the connected peers' info of one specific network
+		// It should be called only when init with NetworkSeparation option
+		ConnectedPeersByNetwork(string) ([]peer.AddrInfo, error)
```

Comment:
it is actually `ConnectedPeersByTopic`

Reply:
`Agent` only export networks instead of topic, topic is a internal concept.

Reply:
i mean,  the name`ByNetwork` is somehow confusing, network may refer to mainnet vs. testnet